### DISCUSSION TITLE
Fastening the Plexiglas panel with the screws/nuts

### DISCUSSION THREAD

**Discussion by haglebu** [thehighnibble/thehighnibble.github.io#141] [IMSAI8080esp replica]
My IMSAI8080esp front panel is finished, now it just needs to be "fixed" correctly. I tested Dave's instructions with the corner reinforcements and was not satisfied with them because the first micro-cracks still appeared. So, what to do.

The problem is that the screws/nuts are supposed to create a compressive stress in the Plexiglas through the tension. However, as the Plexiglas is not 100% in contact with the aluminum housing, tensile stress is generated in the small recess. This definitely leads to a crack in the corner, see picture.

There must therefore be no tensile stress in the small recess. I avoid this simply by placing a small, thin washer between the housing and the Plexiglas. Now there is a direct transfer of force from nut to Plexiglas to washer to housing to screw head. The tensile stress in the screw/nut now only generates a compressive stress between them. That's it. But here too, please tighten the screw with care; especially as far as the two lower opposing ones are concerned.

(I discussed this comment with Dave in advance.)

![IMG_6877](https://github.com/thehighnibble/thehighnibble.github.io/assets/52176362/de2f298e-31d5-4981-ab6f-8e1714c35dea)

![IMG_6880](https://github.com/thehighnibble/thehighnibble.github.io/assets/52176362/cd939487-a3b3-49d0-ab39-78472b5c1e4b)

**Comment by nbreeden2** [comment]
Looks like a good idea. I'll try this on my next build.

**Comment by nbreeden2** [comment]
After seeing this I retrofitted 4 nylon washers on my last build, I've built a number of the systems. The washers worked really well. The result was a nice even gap between the electronics / panel assemble and the metal wrap. I had previously glued on the corner reinforcers and left those in place. My feel was the nylon washers could be slightly deformed. So far, I'm very happy with how it turned out.
